client/src/ingredients/specifications.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export function Specifications({ specifications }) {
+  if (specifications === "non-standard") {
+    return (
+      <>
+        <h2>Specifications</h2>
+        <p>
+          This feature is non-standard and is not included in any official
+          specification. Do not use it on production sites facing the Web: it
+          will not work for every user. There may also be large
+          incompatibilities between implementations and the behavior may change
+          in the future.
+        </p>
+      </>
+    );
+  } else {
```

Comment:
I've done this now. I tested what would happen, but right now it can never reach this case as `build-json` will already be mad at you. So, this can only happen with things that don't come from stumptown-content.